test/e2e/main_test.go

```diff
@@ -70,6 +71,9 @@ func TestMain(m *testing.M) {
 	// the VPC images storage.
 	podvmImage := os.Getenv("TEST_PODVM_IMAGE")
 
+	kbsImage := os.Getenv("TEST_KBS_IMAGE")
```

Comment:
Again this is one of those things where I wonder if this should be part of env var or the properties file.

IIUC anything specific to provider goes in properties and anything generic to the whole test suite goes in env var? But it is less obvious at the first glance for someone new.